docs/gui/menus.md

```diff
@@ -272,12 +272,15 @@ A `MenuProvider` is an interface that contains two methods: `#createMenu`, which
 
 A `MenuProvider` can easily be created using `SimpleMenuProvider`, which takes in a method reference to create the server menu and the title of the menu.
 
+##### Open a menu
```

Comment:
Once again, the section above this still references `NetworkHooks#openScreen`. Please update.

Also, this header is not needed since it's made obvious by the documentation structure.